examples/wasm/mandelbrot/mandelbrot.html

```diff
@@ -1,47 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<meta charset="UTF-8">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>V Mandelbrot WebAssembly Example</title>
-</head>
-<body>
-	<canvas id="canvas" width="200" height="200" style="width:100%;height:100%;image-rendering: crisp-edges;"></canvas>
-	<script>
-		var canvas = document.getElementById("canvas");
-		var ctx = canvas.getContext("2d");
-		var memory;
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>V Mandelbrot WebAssembly Example</title>
+  </head>
+  <body>
+    <canvas
+      id="canvas"
+      width="200"
+      height="200"
+      style="width: 100%; height: 100%; image-rendering: crisp-edges"
+    ></canvas>
+    <script>
+      var canvas = document.getElementById("canvas");
+      var ctx = canvas.getContext("2d");
+      var memory;
 
-		function get_string(ptr, len) {
-			const buf = new Uint8Array(memory.buffer, ptr, len);
-			const str = new TextDecoder("utf8").decode(buf);
+      function get_string(ptr, len) {
```

Comment:
Can the indentation in the form of tabs be retained so as not to impair accessibility?